src/BusinessLogic/Resources/scss/ui-controls.scss

```diff
@@ -0,0 +1,367 @@
+
+    &:hover {
+      color: $color-white;
+      background: $color-blue-darker;
+      box-shadow: none;
+    }
+
+    &:disabled {
+      color: $color-gray;
+      background-color: $color-light;
+      pointer-events: none;
+    }
+  }
+
+  .pl-form-group {
+    position: relative;
+    border: none;
+    width: 100%;
+    align-items: start;
+
+    &:not(:first-child) {
+      margin-top: 15px;
+    }
+
+    label {
+      position: absolute;
+      top: 8px;
+      left: 16px;
+      color: $color-gray;
+      text-transform: uppercase;
+      font-size: 12px;
+    }
+
+    input {
+      background-color: $color-white;
+      border-radius: 0;
+      border: 1px solid $border-color;
+      padding: 21px 16px 13px;
+      color: $input-control-color;
+      width: 100%;
+      margin: 0;
+      box-shadow: unset;
+      line-height: 22px;
+
+      &:focus {
+        border: 1px solid $color-blue;
+        outline: none;
+      }
+
+      &.compact {
+        padding: 13px 16px;
+      }
+    }
+
+    select {
+      height: 50px;
+      border: 1px solid $border-color;
+      line-height: 50px;
+      padding: 7px 0 0 15px;
+      background-color: $color-white;
+      color: $input-control-color;
+      font-size: 12px;
+      width: 100%;
+      -webkit-appearance: none;
+      -moz-appearance: none;
+      appearance: none;
+
+      &:focus {
+        border: 1px solid $color-blue;
+        outline: none;
+      }
+    }
+
+    i {
+      position: absolute;
+      right: 10px;
+      top: 13px;
+      color: $border-color;
+    }
+  }
+
+  input[type=checkbox] {
+    width: 20px;
+    min-width: 20px;
+    height: 20px;
+    -webkit-appearance: none;
+    -moz-appearance: none;
+    appearance: none;
+    border: 1px solid $border-color;
+    position: relative;
+
+    ~ label {
+      margin-left: 10px;
+    }
+
+    &:hover {
+      border-color: $color-blue;
+    }
+
+    &:checked {
+      background-color: $color-blue;
+      border-color: $color-blue;
+
+      &:before {
+        color: $color-white;
+        content: '\2713';
+        position: absolute;
+        top: 4px;
+        left: 4px;
+        font-size: 20px;
+        font-weight: bold;
+      }
+    }
+  }
+
+  .pl-alert-wrapper {
+    position: absolute;
+    width: 100%;
+    padding: 10px;
+  }
+
+  .pl-alert {
+    position: relative;
+    cursor: pointer;
+    padding: .75rem 1.25rem;
+    margin: auto;
+    border: 1px solid transparent;
+    border-radius: .25rem;
+    flex-direction: row;
+
+    &.pl-alert-danger {
+      color: $color-red;
+      background-color: #f8d7da;
+      border-color: #f5c6cb;
+    }
+
+    &.pl-alert-success {
+      color: #155724;
+      background-color: #d4edda;
+      border-color: #c3e6cb;
+    }
+
+    .pl-alert-text {
+      color: inherit;
+    }
+
+    i {
+      color: inherit;
+      margin-left: 20px;
+    }
+  }
+
+  .pl-checkbox {
+    margin-top: 15px;
+    display: flex;
+    flex-direction: row;
+
+    label {
```

Comment:
Codacy found an issue: [Expected selector "#pl-page .pl-checkbox label" to come before selector "#pl-page input[type=checkbox] ~ label" (no-descending-specificity)](https://app.codacy.com/gh/packlink-dev/ecommerce_module_core/pullRequest?prid=5826400)